WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -99,75 +100,71 @@ extension NSNotification.Name {
     ///
     func showHelpCenterIfPossible(from controller: UIViewController, with sourceTag: WordPressSupportSourceTag? = nil) {
 
-        ZendeskUtils.configureViewController(controller)
+        ZendeskUtils.presentInController = controller
 
         // Since user information is not needed to display the Help Center,
         // if a user identity has not been created, create an empty identity.
         if !ZendeskUtils.sharedInstance.haveUserIdentity {
-            let zendeskIdentity = ZDKAnonymousIdentity()
-            ZDKConfig.instance().userIdentity = zendeskIdentity
+            let zendeskIdentity = Identity.createAnonymous()
+            Zendesk.instance?.setIdentity(zendeskIdentity)
```

Comment:
The chances seem remote, but I wonder: would it be worth adding an `if let` against `Zendesk.instance`?

Reply:
Good point. I can add checks to those.

Reply:
Actually, on second thought, there's really nothing to do here. And now I recall this same confusion when v1 was implemented.

1) There will _always_ be an instance. I don't know why it's optional, I've even tried setting it to `nil` but it's not allowed. The instance is initially created with `Zendesk.initialize`. But even if that fails, the instance has a value. 
2) Thankfully Zendesk handles it gracefully and shows a generic error message to the user when something goes wrong.